src/ripple/overlay/impl/ProtocolMessage.h

```diff
@@ -105,6 +127,8 @@ boost::optional<MessageHeader> parseMessageHeader(
 
         for (int i = 0; i != 4; ++i)
             hdr.payload_wire_size = (hdr.payload_wire_size << 8) + *iter++;
+        // clear the compression bits
+        hdr.payload_wire_size &= 0x03FFFFFF;
```

Comment:
The fact that there's a flog like this in the `payload_wire_size` needs to be documented somewhere (it's also possible I missed the comment documenting it).

Reply:
It's covered in the requirements: https://github.com/gregtatcam/rippled-specs/blob/master/0002-protocol-compression/README.md

Reply:
Got it. I'd like to document this in the code as well.

Reply:
Added comments to `setHeader()`.